### PR TITLE
OI-471: Change repo ownership from orchestration-integrations to test-enrichment

### PR DESCRIPTION
**Pull request by wayne-grant**
OI-471: Change repo ownership from orchestration-integrations to test-enrichment in `.github/CODEOWNERS`, `.circleci/config.yml` (snyk circleci context, secrets alerts channel) and `catalog-info.yaml` (github.com/team-slug, owner)

-- 
Generated by [Octopilot](https://github.com/dailymotion-oss/octopilot) [v1.12.15](https://github.com/dailymotion-oss/octopilot/releases/tag/v1.12.15) from https://github.com/snyk/prodsec-tools-v2

### CODE REVIEW

**Review by popas90 — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -196,7 +196,7 @@ workflows:
           name: Scan repository for secrets
           context:
             - snyk-bot-slack
-          channel: team-integrations-alerts
+          channel: snyk-on-snyk-analysis_test-enrichment
```
Comment (KatieArmstr) [inline .circleci/config.yml:199]: this is not a channel 🤔

Comment (wayne-grant) [reply .circleci/config.yml:199]: snyk-on-snyk-analysis_test-enrichment? https://snyk.enterprise.slack.com/archives/C06GW3LLP4H

Comment (KatieArmstr) [reply .circleci/config.yml:199]: oh I see! just it went to our alerts channel before :)

Comment (wayne-grant) [reply .circleci/config.yml:199]: yes, @popas90 and I decided to change it

**Pull request merged**